lib/zendesk_apps_support/validations/svg.rb

```diff
@@ -22,6 +24,19 @@ def call(package)
           end
           errors
         end
+
+        private
+
+        # to ignore the optional XML declaration at the top of a document
+        def strip_declaration(markup)
+          Nokogiri::XML(markup).root.to_s
+        end
+
+        def are_equivalent_sans_declarations(clean_markup, markup)
+          # skip the check if it isn't possible for the markup to contain a declaration
+          return false unless Nokogiri::XML(markup).root.children.length >= 1
```

Comment:
I realise the output of this Nokogiri operation isn't super obvious, but I spent a while [here](https://github.com/sparklemotion/nokogiri/wiki/Cheat-sheet) before I settled on this.

We could just regex for something that looked like `<xml?`... and strip that instead, but while I was working on it, it seemed less brittle to harness the capability of Nokogiri, since it's already imported in here.